### ISSUE TITLE
/jsonld/ is missing compaction/expansion examples

### ISSUE THREAD

**Issue by ProgVal** [codemeta/codemeta.github.io#31]
Hi,

The page https://codemeta.github.io/jsonld/ ends with "Compaction & expansion examples:" but doesn't have any example.

**Comment by cboettig** [comment]
Indeed. There are examples at https://docs.ropensci.org/codemetar/articles/articles/translating-codemeta.html#json-ld-transforms-expansion-and-compaction, these are written in R but are pretty generic, could easily be moved into jsonld-playground.

I'm a bit on the fence about this section though as you've probably gathered from recent posts.  I do think understanding some JSON-LD fundamentals is super useful, though not essential, in being able to do cool things with codemeta, but it's probably another example of scope-creep that is bloating the webpage. We may be better off dropping this entire page and maybe just linking to more official documentation/tutorials for JSON-LD.
